presto-main/src/test/java/com/facebook/presto/geospatial/TestExtractSpatialInnerJoin.java

```diff
@@ -104,15 +104,6 @@ public void testDoesNotFire()
                                         p.values(p.variable("a", GEOMETRY)),
                                         p.values(p.variable("b", GEOMETRY)))))
                 .doesNotFire();
-
-        // We do not support spherical ST_Contains yet
```

Comment:
why is this test removed?

Reply:
A function with this signature doesn't exist. Comment also states that we don't support ST_Contains function with sphere argument types. So now everything is rowexpression vs expression - it will not compile. And it doesn't make sense to test this